frontend/src/pages/History.jsx

```diff
@@ -188,7 +188,7 @@ export const History= () => {
             </div>
 
             <div className="my-option-header">
-              <img className="my-option-line" alt="My option line" src="/image/svg/historyMyOptionLine.svg" />
+              <img className="my-option-line" alt="My option line" src="/image/svg/HistoryMyOptionLine.svg" />
```

Comment:
`historyMyOptionLine.svg` should be renamed according to this change.

Reply:
Done.